### PR TITLE
Bump version: 1.9.0 → 2.0.0: Use the output parameter as a path in any dataset bucket

### PR DESCRIPTION
**Pull request by lgruen**
Related discussion: https://centrepopgen.slack.com/archives/C018KFBCR1C/p1625016039295800

Companion docs update PR: https://github.com/populationgenomics/team-docs/pull/101

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,12 +56,14 @@ analysis-runner \
     --dataset <dataset> \
     --description <description> \
     --access-level <level> \
-    --output-dir gs://<bucket-path> \
+    --output-dir <directory-within-the-bucket> \
```
Comment (illusional) [inline README.md:59]: Isn't this the directory for all buckets?

File: server/cromwell.py [hunk 2]
```diff
@@ -171,11 +171,12 @@ async def cromwell(request):
             job = batch.new_job(name='driver')
             job = prepare_git_job(
                 job=job,
+                dataset=dataset,
                 access_level=access_level,
+                output_dir=output_dir,
```
Comment (illusional) [inline server/cromwell.py:176]:
```
            if access_level == 'test':
                workflow_output_dir = f'gs://cpg-{dataset}-test/{output_dir}'
            else:
                workflow_output_dir = f'gs://cpg-{dataset}-main/{output_dir}'
```

Maybe putting it next to: https://github.com/populationgenomics/analysis-runner/blob/c9450a3783a01c6eb43c21ea1290a456f2e51915/server/cromwell.py#L117-L120

File: README.md [hunk 3]
```diff
@@ -52,12 +52,14 @@ analysis-runner \
...
```
Comment (illusional) [inline README.md:59]: Isn't this directory within _all_ buckets.

File: server/cromwell.py (hunk 2 above)
Comment (lgruen) [reply server/cromwell.py:178]: Thanks, almost had this identical change, but I'll update it with your variable name :).

File: README.md (hunk 1 above)
Comment (lgruen) [reply README.md]: Good point! Changed to `directory-within-bucket`.

**Review by illusional — APPROVED** [review]

**Pull request merged**